src/test/regress/expected/join_gp.out

```diff
@@ -494,8 +494,14 @@ NOTICE:  table "t1" does not exist, skipping
 NOTICE:  table "t2" does not exist, skipping
 NOTICE:  table "t3" does not exist, skipping
 CREATE TABLE t1 (a int, b int, c int);
+NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'a' as the Greenplum Database data distribution key for this table.
```

Comment:
It seems that these changes are redundant. Such lines filtered during difference searching according to the pg_regress init file.

Reply:
When I delete these lines, they show up in the diff file.

Reply:
Unnecessary correction removed from PR